src/aux-common/bots/BotEvents.ts

```diff
@@ -882,6 +883,157 @@ export interface ImageClassifierOptions {
     cameraType?: CameraType;
 }
 
+export interface OpenClassifyImagesAction
```

Comment:
@TroyceGowdy The name `OpenClassifyImagesAction` doesn't properly reflect what the action actually represents. It should be named `ClassifyImagesAction` instead.